source/sourcing_and_processing_data/processing.rst

```diff
@@ -775,15 +775,15 @@ The processor works with masks, it expects
 
 For example, let's assume  you have a temperature sensor that sends and hexadecimal value.
 
-  .. code-block:: JSON
+  .. code-block:: text
 
     hex value : 2C09
 
 This hexadecimal value contains:
 - a decimal value encoded on 2 bytes
 - the sensor status on a bit.
 
-  .. code-block:: JSON
+  .. code-block:: text
```

Comment:
because this is neither JSON nor JS (have a look at the lines below) and cannot be translated to JS or JSON, this is not even pseudo code.